apps/src/weblab/WebLabView.jsx

```diff
@@ -152,6 +163,24 @@ class WebLabView extends React.Component {
               style={iframeStyles}
             />
             {!this.props.isProjectLevel && <CompletionButton />}
+            <BaseDialog
+              isOpen={this.props.shouldShowError}
+              handleClose={this.closeErrorDialog}
+            >
+              <h1>{weblabMsg.uploadError()}</h1>
+              <p>{weblabMsg.errorSavingProject()}</p>
+              <div style={{position: 'relative'}}>
+                <Button
+                  onClick={() => window.location.reload()}
+                  text={msg.reloadPage()}
+                />
+                <Button
```

Comment:
i think we should remove this button -- 1) having 2 orange buttons is visually confusing since orange buttons mean "primary action" for us, and 2) the user can close the dialog with the "X" or by clicking outside of the dialog

Reply:
Fixed so we now have an orange "OK" button and a gray "Reload" button

Reply:
![image](https://user-images.githubusercontent.com/8324574/74867400-74f13f80-5309-11ea-8167-9f081bd68d43.png)